### DISCUSSION TITLE
Possible to connect to Azure App Service?

### DISCUSSION THREAD

**Discussion by BurgerhoutJ** [containrrr/watchtower#689] [Q&A] [answered]
Hi,

I'm running a Ghost blog in Azure App Service and running Watchtower on my Synology in Docker. On the Synology is everything fine. Watchtower is doing his job with the local docker containers.

But is it possible to connect the "on-prem" Watchtower to Azure App Service, so that the Docker image in Azure App Service is updated as well?

**Comment by simonaronsson** [comment, marked as answer]
Hi,

No, sorry. Not to my knowledge. The main reason for this is that Azure App Service doesn't expose any docker socket for watchtower to interact with. In my experience, you'd likely be better off using a step in your CI/devops pipeline.

Best,
Simme
